README.md

```diff
@@ -301,6 +250,7 @@ Usage: ./LINKS [v1.8.7]
 -t  step of sliding window when extracting k-mer pairs from long reads
 (default -t 2, optional)
 	Multiple steps are separated by comma. eg. -t 10,5
+-j  thread count
```

Comment:
This section needs to be updated - ex. what is default threads, version number. Also, if a user followed this would it only run the C++ stage?